# unique robot id/name
string id

# idle pose, yaw angle in degree
geometry_msgs/Pose2D pose

